index.html

```diff
@@ -1,19 +1,134 @@
 <!DOCTYPE html>
```

Comment:
Other than the only one issue I mentioned, I really liked your style of coding. It's logical, clear and neat.